app/services/audit/logger.rb

```diff
@@ -5,7 +5,9 @@ class Logger
     def self.log(listener, user, data = nil)
       Audit::Notification.notify(listener) do |payload|
         payload.user_id = user.id
-        payload.roles = user.roles.pluck(:name)
+        skip_bullet do
+          payload.roles = user.roles.pluck(:name)
+        end
```

Comment:
isn't there a way to skip bullet declaratively without this code?

also Bullet is not enabled in production so this code feels a bit like a patch.

If you look at `config/test.rb` and `config/development.rb` you can see we already have exceptions for some Bullet

Reply:
Thanks! I was able to remove the code altogether, without needing to add any whitelisting. Probably, updated master changed things.